src/include/gcc.h

```diff
@@ -59,21 +59,24 @@
  * accesses will always be in a single cache line.
  *
  * Atomic writes are often associated with memory barriers, implemented by the
- * WT_ACQUIRE_BARRIER and WT_WRITE_BARRIER macros.  WiredTiger's requirement as
+ * WT_ACQUIRE_BARRIER and WT_RELEASE_BARRIER macros.  WiredTiger's requirement as
  * described by the Solaris membar_enter description:
  *
  *	No stores from after the memory barrier will reach visibility and
  *	no loads from after the barrier will be resolved before the lock
  *	acquisition reaches global visibility
  *
- * In other words, the WT_WRITE_BARRIER macro must ensure that memory stores by
- * the processor, made before the WT_WRITE_BARRIER call, be visible to all
+ * In other words, the WT_RELEASE_BARRIER macro must ensure that memory stores by
+ * the processor, made before the WT_RELEASE_BARRIER call, be visible to all
  * processors in the system before any memory stores by the processor, made
- * after the WT_WRITE_BARRIER call, are visible to any processor.  The
+ * after the WT_RELEASE_BARRIER call, are visible to any processor.  The
  * WT_ACQUIRE_BARRIER macro ensures that all loads before the barrier are complete
  * before any loads after the barrier.  The compiler cannot reorder or cache
  * values across a barrier.
  *
+ * The term publish - as it is used in WiredTiger - refers to writing a value to
```

Comment:
Should we mention this?

Reply:
I think its important to codify this somewhere as it is a term we use throughout the code but it isn't ordering related.

Reply:
Why we can't just always say write to a shared memory location or update a shared variable? We then can remove the term publish completely to avoid confusion.